Make.config

```diff
@@ -36,7 +36,7 @@ $(TOP)/Make.config.inc: $(TOP)/Make.config $(TOP)/mk/mono.mk
 
 include $(TOP)/Make.versions
 
-APIDIFF_REFERENCES=https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/d16-8/e30c41de32f64eca655953b17c78c1d9d70249e3/39/package/bundle.zip
+APIDIFF_REFERENCES=https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/d16-8/87a1b18d8890ab76c4c658abd331dfde68d630ab/49/package/bundle.zip
```

Comment:
we might want to compare with the inserted package for xcode12.3
or at least make a note, somewhere, to do it later - once published